packages/lib/package.json

```diff
@@ -118,7 +118,7 @@
     "@mantine/dates": "7.5.2",
     "@mantine/hooks": "7.5.2",
     "@mantine/modals": "7.5.2",
-    "@mantine/next": "7.5.2",
+    "@mantine/next": "6.0.21",
```

Comment:
Do we still need this dependency at all?

Reply:
Yes, it is used in `_document.tsx` in order to support SSR for Mantine components =).

Reply:
I see, but I think that we don't need that anymore. Look at the [Mantine usage with Next.js](https://mantine.dev/guides/next/), it's not mentioned there. Also it's not used in any template from mantine with next.

Reply:
Ah nice!! I didn't realise that they adjusted the SSR implementation in v7, that explains why the version is still on 6 👍 Thanks for the information ℹ️  I change it.